api/alerts.go

```diff
@@ -209,3 +212,30 @@ func (a *Alerts) Delete(viewName, alertName string) error {
 
 	return a.client.Mutate(&mutation, variables)
 }
+
+func mapHumioGraphqlAlertToAlert(input humiographql.Alert) Alert {
+	var queryOwnershipType string
+	switch input.QueryOwnership.QueryOwnershipTypeName {
+	case humiographql.QueryOwnershipTypeNameOrganization:
```

Comment:
Indent?

Reply:
Unfortunately not,

Example: https://go.dev/tour/flowcontrol/9

```
func main() {
	fmt.Print("Go runs on ")
	switch os := runtime.GOOS; os {
	case "darwin":
		fmt.Println("OS X.")
	case "linux":
		fmt.Println("Linux.")
	default:
		// freebsd, openbsd,
		// plan9, windows...
		fmt.Printf("%s.\n", os)
	}
}
```

I'm using gofmt, so I don't have a say in this.

![image](https://github.com/humio/cli/assets/158591046/5256fdaf-5beb-4ba8-b35e-c49fb7222b6a)